### PR TITLE
Allow manager to retrieve plugins from multiple paths

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

File: yantra/manager.py [hunk 1]
```diff
@@ -39,8 +39,18 @@ def _get_modules(self):
         """Discover all modules in the path"""
         plugin_modules = []
 
+        if isinstance(self.plugin_type.path, list):
+           for p in self.plugin_type.path:
+               plugin_modules = self._get_modules_in_path(plugin_modules, p)
```
Comment (RouganStriker) [inline yantra/manager.py]: I think we should be appending to the list here

Comment (RouganStriker) [reply yantra/manager.py]: We might also want to make it a set to remove any potential duplicate modules

**Comment by geoffwhittington** [comment]
I'm not yet committed to this approach. I may re-open when this change becomes necessary

File: yantra/manager.py (hunk 1 above)
Comment (geoffwhittington) [reply yantra/manager.py]: Good idea, I'll keep this in mind if/when I re-open the MR. Thanks @RouganStriker  nice catch

**Pull request closed without merging**